File: Lodjinha/app/src/main/java/freitas/dias/lodjinha/api/model/Category.java
package freitas.dias.lodjinha.api.model;


import com.google.gson.annotations.SerializedName;

public class Category {

    @SerializedName("id")
    private int id;

    @SerializedName("descricao")
    private String description;

    @SerializedName("urlImagem")
    private String urlImage;

    public Category() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getUrlImage() {
        return urlImage;
    }

    public void setUrlImage(String urlImage) {
        this.urlImage = urlImage;
    }
}

File: app/src/main/kotlin/br/com/rbueno/lodjinha/model/Category.kt
package br.com.rbueno.lodjinha.model


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Category(
    @SerializedName("data")
    val categoryItem: List<CategoryItem>
) : Parcelable

@Parcelize
data class CategoryItem(
    @SerializedName("descricao")
    val description: String,
    @SerializedName("id")
    val id: Int,
    @SerializedName("urlImagem")
    val urlImage: String
) : Parcelable


